setup.py

```diff
@@ -20,7 +20,7 @@
     install_requires=[
         "machomachomangler; sys_platform == 'win32'",
         "bindepend; sys_platform == 'win32'",
-        "wheel>=0.32.0",
+        "packaging",
```

Comment:
```suggestion
        "packaging>=20.9",
```

`parse_wheel_filename` was added in 20.9: https://github.com/pypa/packaging/commit/da68002685b0a7d0027e189129d6bde35303a58b